includes/widgets/tabs.php

```diff
@@ -392,31 +393,38 @@ protected function render() {
 		$tabs = $this->get_settings_for_display( 'tabs' );
 
 		$id_int = substr( $this->get_id_int(), 0, 3 );
+
+		$a11y_improvements_experiment = Plugin::$instance->experiments->is_feature_active( 'a11y_improvements' );
+
+		$this->add_render_attribute( 'elementor-tabs', 'class', 'elementor-tabs' );
+
 		?>
-		<div class="elementor-tabs" role="tablist">
-			<div class="elementor-tabs-wrapper">
+		<div <?php echo $this->get_render_attribute_string( 'elementor-tabs' ); ?>>
+			<div class="elementor-tabs-wrapper" role="tablist" >
 				<?php
 				foreach ( $tabs as $index => $item ) :
 					$tab_count = $index + 1;
-
 					$tab_title_setting_key = $this->get_repeater_setting_key( 'tab_title', 'tabs', $index );
+					$tab_title = $a11y_improvements_experiment ? $item['tab_title'] : '<a href="">' . $item['tab_title'] . '</a>';
 
 					$this->add_render_attribute( $tab_title_setting_key, [
 						'id' => 'elementor-tab-title-' . $id_int . $tab_count,
 						'class' => [ 'elementor-tab-title', 'elementor-tab-desktop-title' ],
+						'aria-selected' => 1 == $tab_count ? 'true' : 'false',
```

Comment:
Here and next comparisons .
```suggestion
						'aria-selected' => 1 === $tab_count ? 'true' : 'false',
```

Reply:
Done